Sources_v3/StreamChat/Database/DTOs/UserListQueryDTO.swift

```diff
@@ -12,10 +12,19 @@ class UserListQueryDTO: NSManagedObject {
     /// Serialized `Filter` JSON which can be used in cases the query needs to be repeated.
     @NSManaged var filterJSONData: Data
     
+    /// Indicates if the query should be observed by background workers.
+    @NSManaged var shouldBeObserved: Bool
+    
```

Comment:
OK, one last try: `shouldBeUpdatedInBackgronud` `isEligibleForBackgroundUpdates` ... basically, something else than `shouldBeObserved` because it's too vague imo.